src/ui/components/AnswerOption/AnswerOption.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import AnswerOption from './AnswerOption';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Forms/AnswerOption',
+  component: AnswerOption,
+  tags: ['autodocs'],
+} satisfies Meta<typeof AnswerOption>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Selected: Story = {
+  args: {
+    label: 'Female',
+    value: 'female',
+    selected: true,
+    onChange: () => {},
```

Comment:
The stories pass onChange: () => {} as a no-op. For Storybook it's often better to use the actions addon to surface interactions in the UI, or pass a meaningful stub. Using an empty function hides interaction and reduces debugging capability.

**Sugerencia:**
```typescript
Use Storybook actions: import { action } from '@storybook/addon-actions'; ... onChange: action('onChange') or at least onChange: () => undefined to be explicit. Example:

import { action } from '@storybook/addon-actions';
...
args: { ..., onChange: action('onChange') }
```